src/components/component.ts

```diff
@@ -11,13 +11,17 @@ export namespace Components {
     static CENTER = "center";
   }
 }
+
+export type IResizeHandler = (size: Plottable.Size) => void;
```

Comment:
nit: would prefer to not introduce a new Plottable.Size interface and just pass two width/height numbers directly. Introducing a new interface can cause confusion later on since some parts of the codebase use it while other parts don't